gallery/src/gallery-components/GalleryTiles.tsx

```diff
@@ -23,7 +23,7 @@ export const GalleryTile: FunctionComponent<GalleryTileProps> =
     const galleryTileClasses = classnames('nx-tile-content', className);
 
     return (
-      <div className="nx-tile">
+      <div className="nx-tile gallery-tile">
```

Comment:
Does `gallery-tile` do anything?

Reply:
> Does `gallery-tile` do anything?

No, not anymore, removed: https://github.com/sonatype/sonatype-react-shared-components/pull/40/commits/7e7906866cffd2b3b42535adbfd814f1565028b6